plugins/generic_sso/views/login.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use app\components\UrlHelper;
+use app\models\settings\AntragsgruenApp;
+use yii\helpers\Html;
+
+/**
+ * @var string $backUrl
+ * @var string $providerName
+ * @var string $buttonText
+ * @var string $description
+ */
+
+echo '<section class="loginGenericSso">';
+echo '<h2 class="green">' . Html::encode($providerName) . '</h2>';
+echo '<div class="content row">';
+
+$action = AntragsgruenApp::getInstance()->domainPlain . 'sso-login';
+echo Html::beginForm($action, 'post', ['class' => 'col-sm-4', 'id' => 'ssoLoginForm']);
```

Comment:
col-sm-4 doesn't do anything anymore, as the grid component from bootstrap is not bundled anymore.